docs/index.rst

```diff
@@ -28,6 +26,25 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available IDEs
+-------------
+* `Ethereum Studio <https://live.ether.camp/>`_
+
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
```

Comment:
agreed. We should put the browser-solidity first...frankly it works best. Aside from Mix there's no better place to develop your solidity in. :P Foundation privileges.